Application/db/seven_queries.txt

```diff
@@ -0,0 +1,63 @@
+#All of customers are from München, who don't live in the city district with zip code "80333".
+SELECT*
+FROM customer
+WHERE city = 'München'
+AND zip_code != 80333
+
+
+#This table sums up the total sales per customer
+SELECT fname,lname, sum(price) AS total_sales
+FROM customer a
+JOIN orders b
+ON a.customer_id = b.customer_id
+JOIN order_details c
+ON b.orders_id = c.orders_id
+GROUP BY fname,lname
+
+
+#All ingredients which were imported from the Supplier "Extro AG" with the ID = 3
+SELECT a.ingredient_id, a.ingredient_name, b.supplier_id, b.supplier_name
+FROM ingredient a
+JOIN supplier b
+ON a.supplier_id=b.supplier_id
+WHERE b.supplier_id = '3'
+
+
+#street_name , zip_code, city data of supplier were written as adress
+select supplier_id, supplier_name,
+street_name ||', '|| zip_code ||' '|| city as adress
+from supplier
+
+#How many times was the product ordered?
+SELECT distinct c.product_id,  count(b.quantity) AS ordered_times
+FROM order_details b
+JOIN product c
+ON b.product_id = c.product_id
+GROUP BY c.product_id
+
+#List of the products, which contain ingredient_id = 5
```

Comment:
It would be great if we add number to each query, so you can see how many queries there are. What do you think? :)